### ISSUE TITLE
Courses carousel scrollbar not visible enough, not good for A11Y

### ISSUE THREAD

**Issue by thexpand** [GoogleChrome/web.dev#6881]
**Describe the bug**
The courses carousel scrollbar in the "Learn" page is not visible enough to indicate that it is a carousel. The horizontal scrollbar uses a non-contrasting colors and initially I couldn't find the "Learn CSS" course, as it is the 4th one (you have to scroll right).

**To Reproduce**
Go to https://web.dev/learn/

**Expected behavior**
Either the scrollbar should be visible enough with a good contrast ratio, and/or there should be a clear indication that there are more items in the carousel - probably an arrow?

**Screenshots**
https://imgur.com/a/Br5oVwt

**Comment by MichaelSolati** [comment]
#6614 should have helped slightly by adding arrow buttons, but you're right about the contrast, I'll be looking at that now.

**Comment by hankchizljaw** [comment]
Yeh my bad, I think I messed up the colours recently
